index.html

```diff
@@ -230,5 +230,38 @@ <h3 class="education-title">Skills</h3>
         </div>
         </section>
     </section>
+
+
+        <!-- Footer and contact form -->
+
+
+        <footersection class="form">
```

Comment:
- [OPTIONAL] I think there is a typo here, this tag should be `<section>` or `<footer>`, but I suggest making it `<section>` to follow best practices for semantic HTML5 tags 💯

> Usually the `Footer` contains the social links and the copyright 👍